webpack.config.js

```diff
@@ -55,7 +58,84 @@ if (process.env.TEST_ENV) {
   displayIcons = icons.slice(0, 255);
 }
 
-module.exports = (env, argv) => {
+const pageDescription = `${icons.length} Free SVG icons for popular brands.`,
+  pageTitle = 'Simple Icons',
+  pageUrl = 'https://simpleicons.org',
+  logoUrl = 'https://simpleicons.org/icons/simpleicons.svg';
+
+async function generateStructuredData() {
+  const getSimpleIconsMembers = () => {
+    return new Promise(async (resolve, reject) => {
```

Comment:
I think we should be able to use async/await directly instead of wrapping it in Promises